zerver/tests/test_users.py

```diff
@@ -393,7 +393,7 @@ def test_cache_behavior(self):
             with simulated_empty_cache() as cache_queries:
                 user_profile = self.example_user('hamlet')
 
-        self.assert_length(queries, 1)
+        self.assert_length(queries, 2)
```

Comment:
Ahh, the answer is that this query count went up in an earlier commit!  So merging the first N commits of this branch broke master.  I fixed that in 5acc7506acd5e83a049cbc7faf6e4d39efb7dbee.

Please be careful in the future to make sure that fixes for bugs introduced in a branch end up in the right commit.